app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -85,7 +93,17 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                         return false;
                 }
             });
+			// TODO(aanorbel): Fix change in state(theme change from notification) changing the selected item.
```

Comment:
Is this a TODO for the current PR or for the future?

Reply:
Future work